### ISSUE TITLE
go to definiton support  for class

### ISSUE THREAD

**Issue by raytroop** [imc-trading/svlangserver#13]
I have test it in vim+coc.nvim and vscode. go-to-definition for class neither work.
Is this expected ?

Thanks.

**Comment by kkanhere** [comment]
Yes. Right now none of the verification constructs are well supported.
